python/pyspark/sql/pandas/utils.py

```diff
@@ -40,7 +40,7 @@ def require_minimum_pyarrow_version():
     """ Raise ImportError if minimum version of pyarrow is not installed
     """
     # TODO(HyukjinKwon): Relocate and deduplicate the version specification.
-    minimum_pyarrow_version = "0.15.1"
+    minimum_pyarrow_version = "1.0.0"
```

Comment:
I also tested against 0.17.1, which is still binary compatible, and all tests pass. So it is not necessary to bump this, but is probably a good idea to set a baseline for future support/maintenance

Reply:
+1

Reply:
+1 for Apache Spark 3.1.0.

Reply:
Yup, +1

Reply:
+1 too

Reply:
Hmm, is it good to use 1.0.1 too for pyarrow?

Reply:
I think that [this](https://issues.apache.org/jira/browse/SPARK-32312?focusedCommentId=17190553&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17190553) is why "1.0.0" is used.

Reply:
I used 1.0.0 as a minimum version for pyarrow because the fixes that went into 1.0.1 were not related to usage in Spark, as far as I could tell. There were no fixes for Arrow Java, so I just used 1.0.1 in the pom, since it is the latest.